modules/hcp-dc-mgmt-cluster.adoc

```diff
@@ -6,143 +6,28 @@
 [id="hcp-dc-mgmt-cluster_{context}"]
 = Setting up a management cluster for {hcp} in a disconnected environment
 
-To set up an {product-title} management cluster, you can use dev-scripts, or if you are based on virtual machines, you can use the `kcli` tool. The following instructions are specific to the `kcli` tool.
+To set up an {product-title} management cluster, complete the following steps.
 
-.Procedure
+.Prerequisites
```

Comment:
@jparrill @celebdor - When you are back from PTO, let me know whether these prereqs still apply now that the procedure has been changed.

Reply:
they are still relevant.